GameOfLife/board.h

```diff
@@ -3,13 +3,11 @@
 
 #include <vector>
 
-const int numberOfRow = 30;
-const int numberOfColumn = 30;
-
 class Board
 {
 private:
     std::vector<std::vector<bool>> theBoard;
+    int numberOfCollumn, numberOfRow;
```

Comment:
numberOfColumns & numberOfRows
it should use plural form as there are more than 1